server/nuxeo-nxr-server/src/main/resources/templates/common-base/client/scripts/taskGC.groovy

```diff
@@ -0,0 +1,33 @@
+import org.apache.logging.log4j.LogManager
```

Comment:
There's still an interrogation about committing this script or not.

Reply:
Checked with @tmartins ans he is OK committing the script there 😃